book/guides/data_guide.md

```diff
@@ -17,6 +17,88 @@ To help onboard you to this new way of working, we have written a guide to Files
 
 We recommend you read this thoroughly, especially the part about Git and GitHub. LEAP provides several [cloud buckets](reference.infrastructure.buckets), and the following steps illustrate how to work with data in object storage as opposed to a filesystem.
 
+### Writing Xarray Datasets to Cloud Storage as Zarr
+
+[Xarary](https://docs.xarray.dev/en/stable/getting-started-guide/why-xarray.html) an open-source python project and data model for working with n-dimensional arrays. It is very useful for manipulating large gridded climate datasets.
+
+Below are a few snippets on how to write your Xarray Datasets to cloud storage.
+
+:::\{note}
+Writing to Zarr is the recomened way to save Xarray Datasets. It is very performant and scales incredibly well. That said, there are a few gotchas to look out for. Recently, [Zarr Python V3 was released](https://zarr.dev/blog/zarr-python-3-release/). Depending on which `zarr-python` version you have installed on the hub, writing can look slightly differant!
+
+You can check the version with
+
+```python
+import zarr
+
+print(zarr.__version__)
+```
+
+:::
+
+#### Write to GCS (Google Cloud Storage)
+
+You can write to a GCS bucket with Xarray with a single line of code if you are using the 2i2c Jupyter-hub as the authentication should be ~automagically~ configured.
+
+```python
+import xarray as xr
+
+# Note: We are using an Xarray tutorial dataset in this example
+ds = xr.tutorial.open_dataset("air_temperature", chunks={})
+
+path = "gs://leap-scratch/<YOUR_USERNAME>/<DATASET_NAME.zarr"
+
+
+# optional: you can speciy the Zarr format if you have Zarr python v3 installed.
+zarr_format = 3
```

Comment:
Do we perhaps want to show an alternative path with Zarr v2 in case someone is using an older image for some reason?

Reply:
Sure!